staging/src/k8s.io/cloud-provider/cloud.go

```diff
@@ -49,6 +49,8 @@ type Interface interface {
 	LoadBalancer() (LoadBalancer, bool)
 	// Instances returns an instances interface. Also returns true if the interface is supported, false otherwise.
```

Comment:
Can we add a deprecation note for instances? Would like to minimize how long we need to support both.

Reply:
@andrewsykim proposed only cloud node-controller use instancesV2, I don't think he want deprecate current Instances.